bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/SvgImageCreatorImp.java

```diff
@@ -150,16 +190,17 @@ private boolean generateSvgImage(File imagePresentationDir, UploadedPresentation
 
         if(rasterizeCurrSlide == false) {
             NuProcessBuilder convertPdfToSvg = createConversionProcess("-svg", page, source, destsvg.getAbsolutePath(),
-                    true);
+                    true, convPdfToSvgTimeout);
 
             convertPdfToSvg.setProcessListener(pHandler);
 
             NuProcess process = convertPdfToSvg.start();
             try {
-                process.waitFor(convPdfToSvgTimeout + 1, TimeUnit.SECONDS);
+                process.waitFor(convPdfToSvgTimeout, TimeUnit.SECONDS);

```

Comment:
```suggestion
                process.waitFor(convPdfToSvgTimeout + 1, TimeUnit.SECONDS);
```

I suggest to keep the extra 1 second.